frontend/src/components/ExperimentCollection/ExperimentCollection.tsx

```diff
@@ -34,7 +34,8 @@ const ExperimentCollection = ({ match }: ExperimentCollectionProps) => {
     const nextExperiment = experimentCollection?.next_experiment;
     const displayDashboard = experimentCollection?.dashboard.length;
     const showConsent = experimentCollection?.consent;
-
+    const totalScore = experimentCollection?.total_score
+    const scoreClass = experimentCollection?.score_class
```

Comment:
Isn't the score class defined on the experiment collection's theme's header configuration?